web/yo/app/views/drugs.html

```diff
@@ -0,0 +1,39 @@
+<div class="drugs">
+        <h3 class="headerStyle">Curated Drugs</h3>
+        <table class="table" datatable="ng" dt-options="dtOptions" dt-columns="dtColumns">
+            <thead>
+            <tr class="dashedHeader">
+                <th>Name</th>
+                <th>NCI Thesaurus Code</th>
+                <th>Description</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr data-ng-repeat="drug in drugList">
+                <td ng-include="getTemplate(drug)">
```

Comment:
I'm bit confused here. @victoria34 is this how we switch component? Not using ng-if?

Reply:
@zhx828 Generally we use ng-if to switch components. ng-include will include external html template, which is not suitable in here.

Reply:
@victoria34 great, thanks for confirming. @Yizhuzhu please make the changes

Reply:
I will do it.